index.html

```diff
@@ -1 +1,2 @@
 <h1>First Page</h1>
```

Comment:
```suggestion
<h2>First Page</h2>
```

Let's make this an h2 for better semantics.